code/modules/antagonists/swarmer/swarmer.dm

```diff
@@ -19,11 +19,12 @@
 	death = FALSE
 	roundstart = FALSE
 	flavour_text = {"
+	<b><FONT color='red'>You are to allow other beings to leave peacefully on the shuttle without interference. The station and its resources are your only concern.</font></b>
 	<b>You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate.</b>
```

Comment:
Actually can even highlight the whole "remain in this location to consume and replicate" since that is the main goal. But highlighting remain in this location seems more in-theme of this PR